### ISSUE TITLE
Update the barcode scanning screenshot

### ISSUE THREAD

**Issue by lognaturel** [opendatakit/docs#183]
Barcode scanning is now in portrait and has a shorter, localized string. @adammichaelwood do you think it's worth updating screenshots for this kind of small change?

The screenshot itself is similar enough that it's not a very big deal but I notice that the alt text is very specific and could be misleading (landscape, thin red line).

**Comment by adammichaelwood** [comment]
I think it's worth the change.

**Comment by lognaturel** [comment]
@troublemagnet Great! There's agreement on the way to go and the things to do are clear so you can simply comment with `opendatakit-bot claim` to assign the issue to yourself. 😊

**Comment by troublemagnet** [comment]
@opendatakit-bot claim

**Comment by adammichaelwood** [comment]
FYI -- 
When I made the current screenshot image, I had to fake it -- I literally took an existing phone image from another screenshot into GIMP, deleted the inset image, added a fake barcode and drew the red line.

This is because it was impossible to get a reasonable, live screenshot with the barcode scanner. (As soon as the barcode is in the sights of the scanner, the scan is completed and the image changes.)

You will probably need to do something similar.
